scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -3966,6 +3966,14 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
     }
   }
 
+  def indentedTypeOpt() = {
```

Comment:
It suggests that indentation is always, but type is optional.
`def typeIndentedOpt` ?

Reply:
Fixed